bin/check-invalidation.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+ID=''
+DRYRUN=''
+
+while [ $# -gt 0 ]; do
+  case "$1" in
+    -d)
+      DRYRUN="--dryrun"
+      ;;
+    --id=*)
+      ID="${1#*=}"
+      ;;
+    *)
+      printf "***************************\n"
+      printf "* Error: Invalid argument.*\n"
+      printf "***************************\n"
+      exit 1
+  esac
+  shift
+done
+
+while [ $(aws --profile dev-frontend     \
+  cloudfront get-invalidation         \
+    --distribution-id E1F7ECRVBF3EX2  \
```

Comment:
Should this ID be in the repo?

Reply:
It's locked behind aws permissions, so I don't think it's a problem.  @mcaramello?